src/selectors/entities/preferences.test.js

```diff
@@ -175,6 +181,10 @@ describe('Selectors.Preferences', () => {
     });
 
     describe('get theme', () => {
+        beforeEach(async () => {
+            jest.spyOn(ThemeUtils, 'blendColors').mockReturnValue('#efefef');
```

Comment:
Why do we need this mock? Shouldn't `getTheme` be able to use the theme you've given it in the test case to actually blend it?

Reply:
it requires the function to be mocked to be able to call it and get a value from it otherwise it defaults to undefined.

Reply:
But aren't we just calling it normally? It doesn't hit do anything fancy, so it should always just give us the same result whenever we call it.

Reply:
It's failing without the mock, as far as I saw its taking some properties from the store that aren't there so I either had to mock the function or build up the store.